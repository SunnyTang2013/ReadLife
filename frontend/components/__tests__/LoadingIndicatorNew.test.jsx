AES_ENCRYPTED_FILE
ltjgCdz9te4jKRKIEvyhVWdBQUFBQUJvdkM2NV9wVF9iXzg4V3k1M25jYzJGTGU5empMaFYwS3c0TWkzUWxwdlUtd0dWS1ducUt4S1NoaVJxVXNJcHpjSThhbEJOUm1VUlg5YkpyY1RPUnljMXVvNVFlY2tUUHJVemlDQ1g2YkRRMjlLa2tyNnJiUGxDSUQyaXIweHR3WURyQmpyTjZWMWk4OHNUNUVraFV6M2g1V0tySkZVM0dDRzNZVkxGNUc2dUtaRm0xQlA0OFNRYmxVU1RlVXZ3eGxSRW1hS05sWFR3bWM4MmRwOGdsaVhGWE94SWQwSVNtdG1sRHBRT3c0YTN4bmJfVmxsWld6NU42ak11TDR1bnJlRnhXNFFuYUJOcmtLbURabzFlOFViNTJ6cHYxdWlnU3hzQW5ld290b3VLSW9qcmVpbHZwZTFiZ1k0dnRCdXJCdElvcG0xSXVWMDJ5b1ZVOTM4YjVvWnZ0cU5ja01wa2JOcWhVUjQwQTdsczEySWtCa2dqZWF5SmNweWw0TGFvaWlGaUJpX0dHQjdUV05PenRHeHFydW9OWGZydGRuNWUyRzNUYUtmMUxQQ1BYemJRSzA4blJBeVQzUTdWT3dncW5qTHl1bXRBZ1VjUmU3QklsT0Rxc1o2QTJZYzdoeDNWV0x0elh3Q2VTUGxrLWotcmJYWkt5VGFZMGZ4cmp2NVdRTTJVR0JPdUJiNXVQb0JrS1Fyem92THNya1l5X1o2ei0wUS10djBrUjVuU3Y5ajNaNjV0YmFIZWU3YlNMUWNEa1dZMjlrQl9FZlJWUGRSamY3bWhpSG9MUThHeV9rdTI4MHl0TEtZSk5ibzRrRWw2M085eXlZMmRXYWhjMFYzXzRScDU3ZVM5djhWb0J0WmQ3MFpDN3o5WnhYQzNFdlpBd3d6RjA5V04yX3B3amhyWEJOWXFHNURfam42UW5DSHRjaFQ1STZOMmJzcGJ4RE9ld1FGel9kQjg3WDA5eVNybTVRZFJJak84Z1d3S1RiVFpRVVg3am5LU0toUDVkX3pyM2REMlhZbXpmSGxqaEp2R1ZFaDRjcGo0RXo4UjNDUERacXR5eUpKNG11dWsxRjl2dWpiNFJYWTlmX1VjTW1UUUU4X1R1MnUtSnJ1ZC1vQTFiYjE4YVM2R2JRdTRVbXk1THgxVHJHa0hqcUl4RTAzYWN4ejVNbURzM3d6NUx3dW83LWh0ZnplcU9faW1YcDZNTjIwZkExeEJ1Q21obTFYdHJqdmRrWkRhdVVTOE91WXdjUUJrcXVDMjlKdllJT1lLNTdZc1RkcDJzd2tsYkZsRzkxVjBRMnN4d3MzckhrbVVHX2hDTHJlc2NQS284QWdhbmhwa2FEZnRXVWFuNGhwVnEyTDY3X1VEUHpUNzZjRGRGQWpQWXF2VEVfVERzSDVvVUhObk5vWlZvLS1ROWRRU3BRNGJzWTVoRnU0UEVXc2hrdzNPeF95NVRua0xsTXB6LVhub0w2Wnh4Y0NkNGM1WmV5MHJNa1BIT1BnRDlaQ1FUVFlMcVV0N1dsTmV1SmlvbGtIdF9TTTNUaFp5R2VhU3QtUzQ0MDR4NnRzYWdJVmpsbzh2akhyX0owM3hvdTVTRGhLOVA0YXJPUmV0eFRoVmtqa3UtRjNhUElFb2Nwc3BZdExEeFo1bFYzUzA3V3Z0N1dkMkI3OXdaQ3doaVUycmlRdm5pMTJLczc5dnVqMkVzN0R3LWtPZXUwdExnTkMza3I3RUt6eGJDQzRsY0h0TmRCWGZFNkZURHZfc3IxSFMtT1BxbXptRTkzcVdwRmo1X21hc29tVzB2Nm9ybHVYcUpYLWk5V3kya0ZnUnQ4VzF4bUljS0p3dXVNdUJldXEtQlZKQllQOVhzT0RmUHpqYTVna21UWTJ6VWVIc2I3SEhGdzhyYjd6bHFxUndMdjR2TXpWbTV0bV9lLUM5VG9SUVJfNzFEaUZ1b2taTXZKbng4Y1dhSExWQk5YTGI0MFU4bEFCc3NsbXg5WUJMX0FSQnVCeW42bFo0WlBZSHpUeEgxS1RNdDBZdG9RdXNQQ0hyOE1WWmIzZzRNeEstbDFjVEdDNUtwQTE5aDFrbWpyUEMwOF9hcS11bE5NbnRlMHBVMU03VFFBQ212VmR4bDctWWk0al93ZmVUVzBkTktpYWZhTE02YzdVV0J5dFh0V1hOYUlSVjhlUllIMm5FMnc5UVU5UERRWGxodkZTeWxHckE3X2RnYlZJZ2NBZTVxb01hOU1KRjVZS0JhZnhBWlFtNy1OTGw2TTF3dUljVm8tY0ZrZWJuTTZ1cjhaZDBtNXE4cDNuUnN1Sk1rOUIxTFlRZHYwb3c1M0p4VVBUeXc9PQ==