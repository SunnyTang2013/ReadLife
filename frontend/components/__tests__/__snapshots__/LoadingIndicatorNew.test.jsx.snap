AES_ENCRYPTED_FILE
ltjgCdz9te4jKRKIEvyhVWdBQUFBQUJvdkM2NTQ1WE1kWmtTZ0I2dFR4MjVmaVVUam1BckpwakpQcGdTcFp6clFPOFRNdm8wRlE5NmVlMV8wUzl2dWlDRWhxcEItaGVubEx2R3ZESGR4SE9CdUFEZ3RfbzREZkRwanlVNWJlMWNzQkl2US1mUTZ2Z1RuTVdZOXJKcW5pODdEamxxeEZSd1NZdjJ2MDAxaUt1cW5MLWpRWjBDTE01NEJ5RFMzNzE4VFpUb29mbDVnSTI0NE5ERWI4Y2xkTW5iY3NSTkdnS2hEUVlKMnhFZzcxbVc5b2xUM1cyUzFER0ZGMEZPMUpPbzR3b3U5MUR3QzVMaFdVRVBiSGN4TERocDgzMlJubjhHM0ptcHdndF9HcmF5STRTUkJjVjNBVTN2RHdELXpVNGNTV2lFekVlU0VCVHpMcHRoczFFRXBqSXFTVFpRS2ZqbmVjWXRQcHMwVmdoYkhZaWhXM1J5VmU0LUtiMi1fSTRsaldvanFFZXlIYmhRVVlkV0FaUUxaQndBeGJVZg==